### ISSUE TITLE
ignore_collision not part of obs_dict  variable?

### ISSUE THREAD

**Issue by ShogoAkiyama** [peract/peract#11]
Hello, first of all thank you for sharing the code. amazing work!

I have a question about the `ignore_collision` variable in the replay buffer.

Currently, current_poses `ignore_collision` is inserted  in to the replay memory here
https://github.com/peract/peract/blob/main/agents/c2farm_lingunet_bc/launch_utils.py#L174

Shouldn't the agent predict the next_poses `ignore_collision` and it be inserted to obs_dict instead?

Thank you in advance!

**Comment by MohitShridhar** [comment]
@ShogoAkiyama, `ignore_collision` is used to supervise the agent during training. It's a ground-truth label for activating collision avoidance, and not really part of the "input observation".

**Comment by ShogoAkiyama** [comment]
Thank you for replay!
Sorry for the lack of explanation.

I tried to train a single task of `slide_block_to_target` in PerAct and it resulted in selecting a path to avoid when pressing the block as shown in the video below.
This is probably due to the fact that the data inserted into the replay memory when the block is first pressed during training has `ignore_collision=False`. (The learning loss is blue).

So I changed the code as follows to predict `ignore_collision` in `next_state `when learning.
```
final_obs = {
    ...
    'ignore_collisions': [int(obs_tm1.ignore_collisions)],
}
```
https://github.com/peract/peract/blob/main/agents/c2farm_lingunet_bc/launch_utils.py#L192-L198

This is changed `ignore_collision=True` when pressing block, which is a good result. (The learning loss is pink)

My question is, I think your code predicts ignore_collision in the `current state t`. Shouldn’t it use ignore_collision in the `next state t+1`?

Thank you.

![movie](https://user-images.githubusercontent.com/28297015/212903867-8ad7a7d3-c5b9-476f-b0d2-54fe79a622b7.gif)


![Screenshot at 2023-01-17 18-54-12](https://user-images.githubusercontent.com/28297015/212902954-76a6dffe-dd67-4c27-90a5-f5fd91ade787.png)
